builtin.go

```diff
@@ -266,13 +258,11 @@ func handleWHO(c *Client, e Event) {
 		}
 
 		ident, host, nick, account = e.Params[3], e.Params[4], e.Params[5], e.Params[6]
-		realname = e.Trailing
+		realname = e.Trailing()
 	} else {
 		// Assume RPL_WHOREPLY.
-		ident, host, nick = e.Params[2], e.Params[3], e.Params[5]
-		if len(e.Trailing) > 2 {
-			realname = e.Trailing[2:]
-		}
+		// format: "<client> <channel> <user> <host> <server> <nick> <H|G>[*][@|+] :<hopcount> <real_name>"
+		ident, host, nick, realname = e.Params[2], e.Params[3], e.Params[5], e.Trailing()[2:]
```

Comment:
Not really related to the PR per se, but is hopcount known to always be one character? If not, this could be wrong.

Reply:
That's another TODO item that I'll be fixing before it's merged. I'm not sure why I had it skipping the first two characters (assuming number and space). I _think_ it was that whatever server I was originally testing on, it was always returning 1 even if there were multiple hops, but that is of course making assumptions.

Will fix when I get a chance -- will just strip `^[0-9]+ ` essentially.